academy/3-my-own-chain/run-prod.md

```diff
@@ -1,13 +1,15 @@
 ---
 title: Move to Production
 order: 21
-description: Elements to run your Checkers in production
-tag: deep-dive
+description: Elements to run your checkers in production
+tags:
+  - guided-coding
+  - dev-ops
 ---
 
 # Move to Production
 
-How would you apply what you learned about [running in production](/academy/xl-run-prod/0-overview.html) to your checkers blockchain?
+How would you apply what you learned about [running in production](/hands-on-exercise/5-run-in-prod/0-overview.html) to your checkers blockchain?
```

Comment:
Shall I move the file in this folder right now?

Reply:
It's fine for now. I will move them after the merge. 😉